spec/support/logstash_test.rb

```diff
@@ -0,0 +1,33 @@
+require "stud/temporary"
+module LogStashTest
+  class Certicate
+    attr_reader :ssl_key, :ssl_cert
+
+    def initialize
+      @ssl_cert = Stud::Temporary.pathname("ssl_certificate")
```

Comment:
No change required, just wanted to let you know: I added some PKI support to Flores, so soon we'll be able to generate SSL certificates from Ruby directly for easier testing.